hack/graph-util.py

```diff
@@ -346,7 +346,6 @@ def delete_label(node, label, token, key=None):
     request.get_method = lambda: 'DELETE'
     return urlopen(request)
 
-
```

Comment:
nit: [PEP 8][1]:

> Surround top-level function and class definitions with two blank lines.

Can we keep this one?

[1]: https://www.python.org/dev/peps/pep-0008/#blank-lines